zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.13.1)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(esp-ena)
+
+FILE(GLOB app_sources ../src/*.c*)
+target_sources(app PRIVATE ${app_sources})
```

Comment:
not usable yet. copied from CovdiBracelet, and needs to be fixed (use main/main.c instead of src/*)